acceptance: re-enabled disabled tests

knz opened the pull request:
Based off #15503.

knz committed:
acceptance: re-enabled disabled tests

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/15505)
<!-- Reviewable:end -->

tamird commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/> 

Should be good to rebase now.

---

Reviewed 4 of 4 files at r2.
Review status: 4 of 9 files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/15505#-:-Kiqm0xfxoc4dmzKXB-q:blcye33)*
<!-- Sent from Reviewable.io -->

knz committed:
cli/interactive_tests: deflake/fix test_server_sig

This test was intending to check that if a hard shutdown was initiated
with a double Ctrl+C, the exit code was 130.

Firstly, the test was incorrect: if the server was fast enough to shut
down, the expect script would see the shutdown message but observe a
(correct) exit code of 0, and report (incorrectly) the test as failed.
This patch corrects this behavior by ensuring the value 130 is only
tested if a hard shutdown was actually observed.

Additionally, the test had become broken: because the server had
gotten so good at shutting down fast upon Ctrl+C, the hard shutdown
was much more difficult to observe.  This patch makes the test useful
again by forcing an open client txn, which prevents the fast shutdown
path on the server and thus makes a 2nd Ctrl+C effective.

RaduBerinde commented:
Nice! LGTM
